pandas/core/strings.py

```diff
@@ -1842,23 +1842,38 @@ def extractall(self, pat, flags=0):
         return str_extractall(self._orig, pat, flags=flags)
 
     _shared_docs['find'] = ("""
+    Retrieve least index of substring.
```

Comment:
How about "Returns the lowest index of the substring" which is a bit closer to the stdlib str.find

Reply:
Ya Sure!